ghdocs/Contributing/Contributing.md

```diff
@@ -0,0 +1,92 @@
+# [office-ui-fabric-react](../../../README.md) › Contributing
+
+This document describes how to setup your development environment and contribute changes to the **office-ui-fabric-react** project. This document assumes basic working knowledge with Git and related tools. We are providing instructions specific to this project.
+
+## Setup
+
+- Make sure you have a **github** accont. If not, create one.
+- Install **Node.js LTS 8** or greater from [Node.js website](https://nodejs.org/en/).
+
+## Building
+
+If you do not wish to contribute chages to the **office-ui-fabric-react** project, please follow the instructions on the [README](../README.md) page. Else, keep reading.
+
+## Creating your own fork
+
+If you wish to contribute changes back to the **office-ui-fabric-react** repository, start by creating your own fork of the repository. This is essential. This will keep the number of branches on the main repository to a small count. There are lots of developers in this project and creating lots of branches on the main repository does not scale. In your own fork, you can create as many branches as you like.
+
+- Navigate to **[Github](https://www.github.com)** with a browser and login to your github account. For the sake of this document, lets assume your account is **johndoe**.
+- Navigate to **[Office-ui-fabric-react](https://github.com/OfficeDev/office-ui-fabric-react)** repository in the same browser session.
+- Click on the **fork** button at the top right corner of the page.
+- Create the fork on your user name. Your github profile should now show **office-ui-fabric-react** as one of your repositories.
+- Create a folder on your device and clone your fork of the **office-ui-fabric-react** repository. e.g. https://github.com/**johndoe**/office-ui-fabric-react.git. Notice how your github user name is in the repository location.
+
```

Comment:
Suggest setting up an `upstream` repo that points to `https://github.com/officedev/office-ui-fabric-react.git`.